app/views/content/train-to-be-a-teacher/how-to-apply-for-teacher-training.md

```diff
@@ -88,51 +88,32 @@ You can choose up to 4 teacher training courses for each application.
 
 If you do not find what you were looking for, there are [other ways to train](/train-to-be-a-teacher).
 
-Before you start, it’s particularly important to think about your [teacher training personal statement](/train-to-be-a-teacher/how-to-apply-for-teacher-training#writing-your-teacher-training-personal-statement).
+Before you start, it’s particularly important to think about your [teacher training personal statement](/train-to-be-a-teacher/how-to-apply-for-teacher-training#writing-your-personal-statement).
```

Comment:
```suggestion
Before you start, it’s particularly important to think about your teacher training personal statement.
```
wondering if we need a jumplink here, just cos the info is in the very next section?